### PR TITLE
little changes

### PR DESCRIPTION
**Pull request by mlodic**
* adjusts to urlscan analyzer
* removed unecessary `docker-compose-for-tests-no-angular.yml`
* tweaks in docs
* added `Yara_Scan_Stratosphere` to leverage these [rules](https://github.com/stratosphereips/yara-rules)
* upgraded Speakeasy to last version

### CODE REVIEW

File: env_file_app_travis [hunk 1]
```diff
@@ -39,7 +39,7 @@ GREYNOISE_API_KEY=test
 INTELX_API_KEY=test
 WHOISXMLAPI_KEY=test
 UNPAC_ME_API_KEY=test
-URLSCAN_API_KEY=
+URLSCAN_API_KEY=test
```
Comment (Eshaan7) [inline env_file_app_travis:42]: this should be left blank for the URLscan test to complete successfuly

Comment (mlodic) [reply env_file_app_travis:42]: ah, why?

Comment (Eshaan7) [reply env_file_app_travis:42]: The `URLScan_Search` analyzer works without API key but the submit_result requires API key. If we give dummy api key, then search one fails

Comment (Eshaan7) [reply env_file_app_travis:42]: I commented the `submit_result` variant of URLscan for this reason only and that's why the test is failing ig

Comment (mlodic) [reply env_file_app_travis:42]: I think that this is because it is not mocked `requests.Session.get`

Comment (mlodic) [reply env_file_app_travis:42]: yes, that was the cause. Now it works.

**Pull request merged**